Taskfile.yml

```diff
@@ -10,12 +10,40 @@ tasks:
     cmds:
       - task -l
 
+  brew:
+    preconditions:
+      - sh: brew -v
+        msg: 'This task requires `brew`. Please refer to this documentation: https://brew.sh/'
+
+  pre-commit:
+    desc: Install pre-commit tool
+    internal: true
+    deps: [brew]
+    status:
+      - pre-commit --version
+    cmds:
+      - brew install pre-commit
+
+  install:pre-commit:
```

Comment:
Better prefix from high level to low level, like `pre-commit` and `pre-commit:install` to have proper namespacing

Reply:
Thanks, change done !